site/docs/about.md

```diff
@@ -1,13 +1,11 @@
 ---
 title: About Substrait
 ---
-# Substrait: Cross-Language Serialization for Relational Algebra
```

Comment:
For reviewers, I think mkdocs actually ignores these top-level headers. It's invalid from a markdown perspective and when I build the site locally, this heading is still appended to the end of the title. i.e., the page's HTML `<title>` is still "About Substrait -  Substrait: Cross-Language Serialization for Relational Algebra" like I assume was intended.